ci/tasks/scripts/lint.sh

```diff
@@ -1,10 +1,12 @@
 #!/bin/bash
 
+set -e -u -o pipefail
+
 ./src/ci/tasks/scripts/with-docker.sh
 
 workspace_dir="${PWD}"
-cd src || exit
+cd src
 
 make lint
```

Comment:
does `make -C src lint` not work?

Reply:
I considered this. Given that normal development happens from within the source directory, I'd like to have commands executed in the same manner, so in the event of an issue, it's easier to replicate